bin/runtests

```diff
@@ -371,17 +371,18 @@ function run_bundle_tests(\SplFileInfo $phpunitFile)
         $filesystem->remove($cacheDir);
     }
 
-    putenv('KERNEL_DIR=' . $bundleDir . DIRECTORY_SEPARATOR . get_kernel_dir($phpunitFile->getPathname()));
-
     // see if this bundle uses doctrine orm, and update the schema if so..
     // (we assume that the bundle will purge the fixtures etc.)
     init_bundle($phpunitFile->getPathname());
 
     write_info('Running tests');
 
+    $kernelDirectory = $bundleDir . DIRECTORY_SEPARATOR . get_kernel_dir($phpunitFile->getPathname());
+    putenv('KERNEL_DIR=' . $kernelDirectory);
+    $_SERVER['KERNEL_DIR'] = $kernelDirectory;
```

Comment:
That's the solution, we need both lines in order to work correctly with the PHP version conditional in the [Symfony Process Component](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Process/Process.php#L1725)